### PR TITLE
Adding text and images to the Registration Method Examples

### LINKED ISSUE #1864: Update the Image Registration Method examples to contain images and useful text output

### PR DESCRIPTION
**Pull request by mbopfNIH**
Closes #1864

### CODE REVIEW

**Comment by mbopfNIH** [comment]
In addition to adding images & output text to RegistrationMethod1 Example, this PR also includes some changes to the conf.py and run_example code to move the modification of sys.path to the conf.py, which seems cleaner. It also uses Sphinx ..Figure:: directives instead of ..Image:: directives which allows for nicer Image captions. Standardized a number of other Examples, also.

**Comment by mbopfNIH** [comment]
Screenshots of RegistrationMethod1. A scrolled or collapsable window for the text would be nice...

Screen Shot 2023-02-07 at 2 36 30 PM
Screen Shot 2023-02-07 at 2 37 51 PM

**Comment by zivy** [comment]
Can the figures go in a horizontal table? Not sure if this is possible, but someone suggested using [list-table](https://sublime-and-sphinx-guide.readthedocs.io/en/latest/tables.html#list-table-directive) to do this with images ([see here](https://github.com/sphinx-doc/sphinx/issues/4003#issuecomment-502025066)). Possibly check if that helps?

**Comment by mbopfNIH** [comment]
Added the sphinx_tools.collapse extension and applied it to DicomPrintTags and RegistrationMethod1 so user can collapse the large blocks of text. See image below.

NOTE: I did not squash this merge since I thought the comment was useful. I could squash it and rewrite the comment instead.

Screen Shot 2023-02-07 at 4 44 51 PM

File: Examples/DicomImagePrintTags/Documentation.rst [hunk 1]
```diff
@@ -6,16 +6,37 @@ Read Image Meta-Data Dictionary and Print
 Overview
 --------
 
-This example illustrates how to read only an image's information and meta-data dictionary without loading the pixel content via the `ImageFileReader <https://simpleitk.org/doxygen/latest/html/classitk_1_1simple_1_1ImageFileReader.html>`_.
+This example illustrates how to read only an image's information and meta-data
+dictionary without loading the pixel content via the
+`ImageFileReader <https://simpleitk.org/doxygen/latest/html/classitk_1_1simple_1_1ImageFileReader.html>`_.
```
Comment (blowekamp) [inline Examples/DicomImagePrintTags/Documentation.rst:12]: This looks like some auto formatting happened here. This should not be in the same commit at the other changes.

File: docs/source/example_utils.py [hunk 2]
```diff
@@ -43,12 +43,12 @@ def run_example(module_name: str, func_name: str, args: list) -> dict:
...
     """
 
-    sys.path.append(f'../../Examples/' + module_name)
     try:
-        example_module = importlib.import_module(f"{module_name}")
+        example_module = importlib.import_module(f"{module_name}.{module_name}")
     except (ImportError, AttributeError) as ex:
         raise ValueError(f"Unknown module: {module_name}") from ex
 
+    print(f'example_module.__name__ = {example_module.__name__}')
```
Comment (blowekamp) [inline docs/source/example_utils.py:51]: Is this a left over debugging message?

**Review by blowekamp — APPROVED** [review]
The output looks great.

A couple small comments to clean up the commit.

File: docs/source/example_utils.py (hunk 2 above)
Comment (mbopfNIH) [reply docs/source/example_utils.py:51]: Yeah, debug output - I'll take it out.

**Pull request merged**